Sources/BuyMeACoffee/BMCColor.swift

```diff
@@ -7,6 +7,21 @@
 
 #if canImport(UIKit)
 import UIKit
+#if canImport(SwiftUI)
+import SwiftUI
+#endif
+
+protocol CustomColor { }
+
+extension UIColor: CustomColor { }
```

Comment:
These empty protocols are really not a good way to handle the SwiftUI/UIKit thing